### PR TITLE
boards: riscv: adp_xc7k_ae350: update dts for atmel eeprom driver

### PR DESCRIPTION
**Pull request by wtlee1995**
Update dts for usage of atmel eeprom driver.

Signed-off-by: Wei-Tai Lee <wtlee@andestech.com>

### CODE REVIEW

File: boards/riscv/adp_xc7k_ae350/adp_xc7k_ae350.dts [hunk 1]
```diff
@@ -19,6 +19,7 @@
 		led0 = &seg7_led1_g;
 		led1 = &seg7_led2_g;
 		sw0 = &user_button1;
+		eeprom-0= &eeprom;
```
Comment (fkokosinski) [inline boards/riscv/adp_xc7k_ae350/adp_xc7k_ae350.dts:22]: Nitpick: add missing space before `=`, i.e. `eeprom-0 = &eeprom;`

**Review by fkokosinski — COMMENTED** [review]
Only one style-related nitpick. LGTM otherwise.

**Review by carlocaione — DISMISSED** [review]

File: boards/riscv/adp_xc7k_ae350/adp_xc7k_ae350.dts (hunk 1 above)
Comment (wtlee1995) [reply boards/riscv/adp_xc7k_ae350/adp_xc7k_ae350.dts]: Thanks, fixed it.

**Review by fkokosinski — DISMISSED** [review]

**Review by carlocaione — DISMISSED** [review]

**Comment by carlescufi** [comment]
@wtlee1995 please rebase

**Review by fkokosinski — APPROVED** [review]

**Review by carlocaione — APPROVED** [review]

**Pull request merged**